src/main/java/com/bravo/user/controller/UserController.java

```diff
@@ -35,6 +35,18 @@ public UserController(UserService userService, UserValidator userValidator) {
     this.userValidator = userValidator;
   }
 
+  @GetMapping(value = "/{name}")
```

Comment:
I would expect "retrieve" in the name here to follow the conventions of the app.

Reply:
The retrieve path is already being used to pull records by ID. Additionally I think it is more common to have a path such as:
/user/{name} so you can expand on other attributes to fetch about that user, such as /user/{name}/group, or some variation of this pattern, which I have also seen OKTA use when integrating with their APIs.